blog/2024-03-12-reductstore-v1_9_0-released/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "ReductStore v1.9.0 Released"
+description: This release, version 1.9.0, introduces several key improvements and features to enhance the overall performance and user experience. These updates include optimizations for disk space management, the inclusion of replication support in the Web Console, and the provision of license information in the HTTP API.
+authors: alexey
+tags: [news]
+slug: news/reductstore-9-released
+date: 2024-03-12
+---
+
+We are pleased to announce the release of the latest minor version of [**ReductStore**](https://www.reduct.store/), [**1.9.0**](https://github.com/reductstore/reductstore/releases/tag/v1.9.0). ReductStore is a time series database designed for storing and managing large amounts of blob data.
+
+To download the latest released version, please visit our [**Download Page**](https://www.reduct.store/download).
+
+## What’s New in 1.9.0?
+
+This release, version 1.9.0, introduces several key improvements and features to enhance the overall performance and user experience. These updates include optimizations for disk space management, the inclusion of replication support in the Web Console, and the provision of license information in the HTTP API.
+
+<!--more-->
+{/* truncate */}
+
+### Disk Space Management and Optimization in ReductStore
+
+ReductStore ensures that you don't exceed your disk space with the FIFO bucket quota. It removes old data when the quota is reached. If it's not possible to free up enough disk space for incoming data, the system rejects the write request. To perform efficiently, it calculates disk usage at startup. This calculation can be slow for large amounts of data, like hundreds of terabytes. However, it has now been significantly optimized and parallelized.
```

Comment:
Possible to separate the two different concepts (Disk Space Management and Optimization in ReductStore)? The transition between the two is no so clear. 
1. ReductStore ensures that you.. 
2. To perform efficiently,...

Reply:
changed:

To keep the quota efficiently, ...